.github/workflows/sanity/setup-apache.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
```

Comment:
```suggestion

set -e
```

Make sure the script crashed on error